src/EVM/TTY.hs

```diff
@@ -863,7 +864,8 @@ currentSrcMap dapp vm = do
 drawStackPane :: UiVmState -> UiWidget
 drawStackPane ui =
   let
-    gasText = showWordExact (num ui.vm.state.gas)
+    gasText = T.pack $ showHex ui.vm.state.gas ""
+    --gasText = showWordExact (num ui.vm.state.gas)
```

Comment:
is this still needed?

Reply:
Oops, this was a little change I made to help debug, it shouldn't be merged. Thanks for catching :)